src/map/mob.cpp

```diff
@@ -1084,11 +1084,12 @@ int mob_setdelayspawn(struct mob_data *md)
 int mob_count_sub(struct block_list *bl, va_list ap) {
```

Comment:
Please apply tabs instead of spaces to the whole function, if you already touch it.

Also please apply usage of the ARRAYLENGTH macro, if we already touch it.

Reply:
This goes against your philosophy to never change formatting of cells you didn't change! I left it untouched just for you.